lang/繁體中文.json

```diff
@@ -137,9 +137,15 @@
                 "name": "基本設定",
                 "autostart": "開機時自動啟動：",
                 "tray": "開機時以最小化自動啟動：",
-                "priority_only": "僅參與優先掉寶遊戲：",
+                "tray_notifications": "啟用系統列通知：",
+                "priority_mode": "掉寶模式：",
                 "proxy": "代理伺服器（需要重新啟動）："
             },
+            "priority_modes": {
+                "priority_only": "僅參與優先掉寶遊戲",
+                "ending_soonest": "優先觀看即將結束的掉寶活動",
+                "low_availability": "優先觀看可用頻道較少的掉寶活動"
+	    },
```

Comment:
Missed this one.

```suggestion
            },
```

Reply:
I thought this was Github automatically use Tab space when I hit Enter, I'll change it.